mu-plugins/central-hub/tests/phpunit/unit/meta-data/isOkayToSaveMetaBox.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Tests for the function is_okay_to_save_meta_box().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\is_okay_to_save_meta_box;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_RegisterMetaBoxes
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_IsOkayToSaveMetaBox extends Test_Case {
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
+	/**
+	 * Test is_okay_to_save_meta_box() should check the meta box key exists in the $_POST array.
+	 */
+	public function test_should_return_false_when_meta_box_key_is_not_in_POST() {
+		$meta_box_key = 'members';
+		$_POST        = [
+			'post_ID'           => '1322',
+			'post_status'       => 'publish',
+			'events'            => [
+				'event-date' => '09-24-2019',
+				'event-time' => '17:30',
+				'venue-name' => 'Ladue Chapel',
+			],
+			'events_nonce_name' => '04c923a557',
+		];
+
+		$this->assertFalse( array_key_exists( $meta_box_key, $_POST ) );
+		$this->assertFalse( is_okay_to_save_meta_box( 'members' ) );
+	}
+	
+	/*
+	 * Test should return false when meta box nonce name key is not set in $_POST.
+	 */
+	public function test_should_return_false_when_meta_box_nonce_name_key_is_not_set_in_POST() {
+		$nonce_name = 'events_nonce_name';
+		$_POST      = [
+			'post_ID'     => '1322',
+			'post_status' => 'publish',
+			'_wp_nonce'   => '04c923a55d',
+		];
+
+		$this->assertArrayNotHasKey( $nonce_name, $_POST );
```

Comment:
@rgadon107 Put the string here instead to improve the readability of this assertion.

Reply:
Done.